corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -612,9 +612,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                                     markers.forEach(m => m.setIcon(locationIcon));
                                     marker.setIcon(selectedLocationIcon);
 
+                                    // Offset by breadcrumbs w/w/o small screen map
+                                    const scrollTopOffset = this.smallScreenEnabled ? 300 : 50;
```

Comment:
what's the 300 constant? Would be nice to either have a comment or define as a constant

Reply:
It's the height of the map plus breadcrumbs (+/- a few px because the breadcrumb bar resizes based on content)

Reply:
I think the comment is fine. It feels a bit fragile, but I also see this strategy was here before your changes so what you did makes sense.

Is the `scrollTopOffset` to define the top "sticky" positioning so it's relative to the smaller parent container?

Reply:
I tweaked this a little in https://github.com/dimagi/commcare-hq/pull/33351/commits/f9dba50e3e2122f657fac12037beabc9d4c99229. It still has our nice magic number for the breadcrumbs (slightly more precise now) but does get the actual size of the map. This might need further updating when we get to scrolling under a fullscreen map, but that's a problem for a different ticket.